src/main/java/de/zalando/ep/zalenium/dashboard/remote/RemoteDriverLogDashboard.java

```diff
@@ -0,0 +1,44 @@
+package de.zalando.ep.zalenium.dashboard.remote;
+
+import com.google.gson.JsonObject;
+import de.zalando.ep.zalenium.dashboard.TestInformation;
+import org.apache.http.entity.ContentType;
+
+import java.io.FileInputStream;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class RemoteDriverLogDashboard extends RemoteDashboard {
+
+    @Override
+    public void updateDashboard(TestInformation testInformation) throws Exception {
+        if (null == testInformation) throw new IllegalArgumentException("testInformation");
+
+        if(!isEnabled()) {
+            return;
+        }
+
+        List<FormField> fields = new ArrayList<>();
+
+        fields.add(new FormFile() {{
```

Comment:
I don't think we want the "double-brace initialisation" anywhere, as it will probably create a garbage collection overhead, as well as potentially leaking memory.

Since this called after every test, it has more chance of being a problem.

I didn't even know that this "double-brace initialisation" was even a thing, until codacy picked it up: https://app.codacy.com/app/diemol_zalenium/zalenium/pullRequest?bid=4117327&prid=1752756#newIssuesView

I did a little digging and came across this article explaining why it's not a great idea: https://blog.jooq.org/2014/12/08/dont-be-clever-the-double-curly-braces-anti-pattern/